wp-parsely.php

```diff
@@ -257,3 +310,21 @@ function maybe_disable_some_features() {
 		unregister_widget( 'Parsely_Recommended_Widget' );
 	}
 }
+
+/**
+ * Enum which represent all options to integrate `wp-parsely`.
+ */
+// phpcs:ignore Generic.Files.OneObjectStructurePerFile.MultipleFound
+abstract class Parsely_Integration_Type {
+	const ENABLED_MUPLUGINS_FILTER        = 'ENABLED_MUPLUGINS_FILTER';
+	const ENABLED_MUPLUGINS_SILENT_OPTION = 'ENABLED_MUPLUGINS_SILENT_OPTION';
+
+	const DISABLED_MUPLUGINS_FILTER        = 'DISABLED_MUPLUGINS_FILTER';
+	const DISABLED_MUPLUGINS_SILENT_OPTION = 'DISABLED_MUPLUGINS_SILENT_OPTION';
```

Comment:
Since the string value of these is changing (in a good way), we'll need to make sure that other systems are aware of the change. Most notably, we may need to update the reports in Gainsight to check the new values once they are syncing over.